test/e2e/cluster.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# shellcheck source=/dev/null
+source "$(dirname "$0")/utils.sh"
+
+cd_root_path
+
+# Build
+make container-build
+
+# Prepare containers
+trap "docker rm -f etcdadm-{0,1,2};rm -f ${PWD}/bin/ca.*" EXIT
```

Comment:
Why does the script use `${PWD}/bin/`? The directory may not exist.

I think the script can create (and remove) its own temporary directory. What do you think?

Reply:
`${PWD}/bin` save the `init` container CA certs, other member need CA certs when join. The `init` container mount etcdadm root dir, so i use the `bin` directory save the CA certs. 

>  I think the script can create (and remove) its own temporary directory. What do you think?

You're right, did we use `docker volume create` to create temporary directory that save CA certs?